controller/appcontroller.go

```diff
@@ -421,7 +421,7 @@ func (ctrl *ApplicationController) handleObjectUpdated(managedByApp map[string]b
 			"kind":         ref.Kind,
 			"server":       app.Spec.Destination.Server,
 			"cluster-name": app.Spec.Destination.Name,
-		}).Debug("Requesting app refresh caused by object update")
+		}).Info("Requesting app refresh caused by object update")
```

Comment:
This change may increase amount of logs dramatically and it may affect cost of logging systems, not sure that it is good idea

Reply:
Agreed. We made this debug level very intentionally because it can be so noisy.

Reply:
In self-service deployments, (eg: using git generators) is easy to deploy something that request a refresh constantly (eg: changing annotations in a configmap) and it would be really hard to identify it without Debug log level (that is really verbose).

My main concern then is, or you have debug mode on (with all logs) or nothing. Or there is another way?
Thanks!

Reply:
> Or there is another way?

There's currently no other way. But I think debug should be okay.... you can turn it on for ten minutes and usually have enough logs to find the problem.

Reply:
But then we need a degradation of the whole system as the trigger to turn on the debug Level 
It is not possible to configure an active monitoring (based on this logs) to prevent it 


What I want to say then, maybe the log info is not a good idea but we need a way to observe this situation, maybe with a metric? Wdyt?

Reply:
Metric sounds like a great idea! Much lighter-weight, but enough to track when there's a spike in refreshes.

Reply:
Nice, then let me close this PR and the move the proposal here https://github.com/argoproj/argo-cd/issues/17122 ok?
Thanks a lot!

Reply:
I think the existing issue should do fine. You bet!